frontend/src/App.vue

```diff
@@ -288,16 +288,26 @@ export default {
     });
   },
   methods: {
-    logout() {
-      this.$axios
-        .post(`${this.$apiBaseUrl}/logout`)
-        .then(() => {
-          location.reload();
-        })
-        .catch((e) => {
-          console.error(e);
+     async logout() {
+        let redirectLocation;
+        await this.$axios.get(`${this.$apiBaseUrl}/logout`)
```

Comment:
I think this might be related to the redirect issue: https://stackoverflow.com/questions/64713413/keycloak-cors-issue-on-logout-redirect As I understand it the whole browser needs to be redirected instead of doing a background request

Reply:
Have changed the logout procedure in VUE. Immediately telling the browser to call our logout procedure. Redirects are handled by the browser.
However, the current local user flow doesn't work properly by having the browser call for logout, so we need two different handlers. If the Micronaut logout allows GET, then have the browser do the work, if the logout does not support GET, then call AXIOS.  The configuration for security-keycloak has GET allowed, the security for Local does not.

Reply:
Sorry for being so persistent, but the only thing that is missing in the standard flow is to set `micronaut.security.redirect.logout: /user/signin` and `get-allowed: true` Then the logout handler in the UI can be used for both methods.

Reply:
Be persistent! I've got local and Keycloak set up correctly now, and we can just let the single `GET /logout` work in both cases. And we can remove that extra RuntimeConfig. The fewer special tweaks we have in our code is for the better. So this is nice that we can rely completely on Micronaut.